documentation/modules/post/multi/gather/chrome_cookies.md

```diff
@@ -1,16 +1,20 @@
 ## Gather Chrome Cookies
 
-Uses [Headless Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome) and [Chrome's Remote Debugging](https://chromedevtools.github.io/devtools-protocol/) to read all cookies from the Default Chrome profile of the user.
+Reads all cookies from the Default Chrome Profile on the target machine. Uses [Headless Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome) and [Chrome's Remote Debugging](https://chromedevtools.github.io/devtools-protocol/).
 
 ## Opsec
 
-This writes to disk temporarily. You may want to consider the tradeoff between getting the user's Chrome cookies and the noisiness of writing to disk.
+### Disk writes
+This writes randomly-named files to disk temporarily. You may want to consider the tradeoff between getting the user's Chrome cookies and the noisiness of writing to disk.
 
 The module writes a random 10-15 character file containing HTML to a directory you can specify via `WRITABLE_DIR`.
 
+### Running processes
+On non-meterpreter sessions, a headless Chrome process will be left running after module execution is completed. You can still find and kill this process manually after the module execution is completed.
```

Comment:
The behavior was changed for killing the chrome process. The chrome process is only stopped when the module runs over a Windows meterpreter session. The kill command on OSX and Linux would stop the meterpreter process for some reason and the module would fail to complete.